### DISCUSSION TITLE
Chromium a une fonctionnalité de surlignement de textes

### DISCUSSION THREAD

**Discussion by jibe-b** [CaptainFact/captain-fact#204] [Ideas]
Si jamais ça peut servir, il suffit de placer ceci

#:~:text=<Les%20mots%20que%20vous%20voulez%20surligner>

à la fin de l'URL

**Comment by vguen** [comment]
Le code source de cette fonctionnalité est disponible ici https://github.com/GoogleChromeLabs/link-to-text-fragment. Il est intéressant de noter que les mainteneurs ont créé des extensions pour Firefox, Safari, Chrome et Edge.

**Comment by Betree** [comment]
C'est plutôt cool ! Ça serait génial que ça se standardise. À prendre en compte pour #191.

Je déplace vers https://github.com/CaptainFact/captain-fact/discussions puisqu'il n'y a pas de concrétisation claire pour l'instant.
